klimenko/Lesson3_4_Klimenko/src/main/java/controller/MVCUser.java

```diff
@@ -0,0 +1,108 @@
+package controller;
+
+
+import dao.ApartmentsDAO;
+import dao.UserDAO;
+import entity.Apartment;
+import entity.User;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.ModelAndView;
+
+import javax.jws.soap.SOAPBinding;
+import javax.servlet.http.HttpSession;
+import javax.validation.Valid;
+
+
+import java.io.UnsupportedEncodingException;
+
+import static service.Cipher.encodeString;
+
+@Controller
+public class MVCUser {
```

Comment:
I think MVCUser is unreadable, better to name it UserController, or smth like that